swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Settings.swift

```diff
@@ -31,15 +31,14 @@ struct AdvancedSettings: Equatable {
         authBaseURLString: "https://app.firez.one/",
         apiURLString: "wss://api.firez.one/",
         connlibLogFilterString:
-          "connlib_client_apple=debug,firezone_tunnel=trace,phoenix_channel=debug,connlib_shared=debug,connlib_client_shared=debug,info"
+          "firezone_tunnel=trace,phoenix_channel=debug,connlib_shared=debug,connlib_client_shared=debug,str0m=info,debug"
```

Comment:
If you are setting an overall `debug` level, you could remove the others. I've found that too noise though because of the TLS stuff with the websocket.

I usually use overall `info` and narrow them down to debug / trace for others.

Reply:
👍

Reply:
So the issue is that only the global filter applies to this glue module. Should we start at `debug` for dev builds and tune out the noisy ones with `info` on an adhoc basis?

That might be better than logging `info` in order to see things during development, because those might end up accidentally making their way to release.

Reply:
Refs #3618